### PR TITLE
docs: fix dead CSRF link

### PR DESCRIPTION
**Pull request by spixy**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by spixy** [comment]
@googlebot I signed it!

**Review by gkalpak — APPROVED** [review]
Thx!

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.

File: aio/content/guide/security.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ That means only your application can read this cookie token and set the custom h
 Angular's `HttpClient` has built-in support for the client-side half of this technique. Read about it more in the [HttpClient guide](/guide/http#security-xsrf-protection).
 
 For information about CSRF at the Open Web Application Security Project (OWASP), see
-<a href="https://www.owasp.org/index.php/Cross-Site_Request_Forgery_%28CSRF%29">Cross-Site Request Forgery (CSRF)</a> and
+<a href="https://owasp.org/www-community/attacks/csrf">Cross-Site Request Forgery (CSRF)</a> and
 <a href="https://www.owasp.org/index.php/CSRF_Prevention_Cheat_Sheet">Cross-Site Request Forgery (CSRF) Prevention Cheat Sheet</a>.
```
Comment (gkalpak) [inline aio/content/guide/security.md]: This link also seems to be dead (and there is no replacement afaik). Could you remove this as well?

Comment (spixy) [reply aio/content/guide/security.md]: substitute with https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html ?

Comment (gkalpak) [reply aio/content/guide/security.md]: Oh, nice! I couldn't find a cheatsheet URL when I looked. Sure, replace with that :+1:

Comment (spixy) [reply aio/content/guide/security.md]: done 👍

File: aio/content/guide/security.md [hunk 2]
```diff
@@ -279,8 +279,8 @@ That means only your application can read this cookie token and set the custom h
 Angular's `HttpClient` has built-in support for the client-side half of this technique. Read about it more in the [HttpClient guide](/guide/http#security-xsrf-protection).
 
 For information about CSRF at the Open Web Application Security Project (OWASP), see
-<a href="https://www.owasp.org/index.php/Cross-Site_Request_Forgery_%28CSRF%29">Cross-Site Request Forgery (CSRF)</a> and
-<a href="https://www.owasp.org/index.php/CSRF_Prevention_Cheat_Sheet">Cross-Site Request Forgery (CSRF) Prevention Cheat Sheet</a>.
+<a href="https://owasp.org/www-community/attacks/csrf">Cross-Site Request Forgery (CSRF)</a> and
+<a href="https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html">Cross-Site Request Forgery (CSRF) Prevention Cheat Sheet</a>.
```
Comment (gkalpak) [inline aio/content/guide/security.md]: While you're at it, maybe you could change the URL to Markdown too 🙏

```suggestion
[Cross-Site Request Forgery (CSRF)](https://owasp.org/www-community/attacks/csrf) and
[Cross-Site Request Forgery (CSRF) Prevention Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html).
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx for making the changes, @spixy :+1:
I made one more suggestion; otherwise lgtm :tada:

You also need to rebase this on master (to make CI happy). While doing so, please also squash the commits into one (to remove unnecessary noise in git commit history).

File: aio/content/guide/security.md (hunk 2 above)
Comment (spixy) [reply aio/content/guide/security.md]: moved to markdown format, squash rebased to master

Comment (gkalpak) [reply aio/content/guide/security.md]: Actually, this doesn't seem to be rebased on latest master nor have the commits squashed into one 😕

Comment (spixy) [reply aio/content/guide/security.md]: 1a12d7c42ad60c6f865e097b1af62887286c1b29

but I also need to do that also in `patch-1` right?

**Review by mhevery — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]

**Review by mhevery — APPROVED** [review]

**Review by mhevery — APPROVED** [review]
reviewed-for: global-approvers

**Review by IgorMinar — APPROVED** [review]
LGTM. Thanks for fixing this!

Reviewed-for: fw-security

**Comment by mhevery** [comment]
merge-assistance: It seems that Github is confused and added @jelbourn twice to the reviewers, and it does not seem to be possible to remove them. The PR is ready to merge as pull approve is ✅ .

**Pull request closed without merging**